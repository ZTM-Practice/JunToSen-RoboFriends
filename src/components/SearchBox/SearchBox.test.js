import { shallow } from "enzyme";
import React from "react";
import SearchBox from "./SearchBox";

describe('SearchBox Component', () => {
    it('it renders the SearchBox component', () => {
        const mockOnChange = jest.fn();
        expect(shallow(<SearchBox searchChange={mockOnChange} />)).toMatchSnapshot();
    });
});
