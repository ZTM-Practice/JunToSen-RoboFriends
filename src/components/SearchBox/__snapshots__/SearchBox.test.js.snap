// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox Component it renders the SearchBox component 1`] = `ShallowWrapper {}`;
